logs/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine
+FROM cloudcourseproject_base
```

Comment:
Yes, but per requirements, both containers (web server and logs collector) should be built on top of base image